task_processing/plugins/kubernetes/task_config.py

```diff
@@ -118,6 +120,22 @@ def _valid_volumes(volumes: Sequence["DockerVolume"]) -> Tuple[bool, Optional[st
     return (True, None)
 
 
+def _valid_empty_volumes(volumes: Sequence[EmptyVolume]) -> Tuple[bool, Optional[str]]:
+    for volume in volumes:
+        if set(volume.keys()) != VALID_EMPTY_VOLUME_KEYS:
+            return (
+                False,
+                f'Invalid empty volume format, must only contain following keys: '
+                f'{VALID_EMPTY_VOLUME_KEYS}, got: {volume.keys()}'
+            )
+        if volume.get('medium') not in [None, "Memory"]:
```

Comment:
might be nice to make the valid mediums into a set and extract it to a named constant (like `VALID_DOCKER_VOLUME_MODES`)